### ISSUE TITLE
Amazing App. Is it active

### ISSUE THREAD

**Issue by anandbhaskaran** [HeroBlackInk/ultimate-todoist-sync-for-obsidian#84]
I am a heavy Obsidian task user and this plugin made my day! This plugin is better than all the other plugins for Todoist sync.

However, is this project active? Will there be new releases?

**Comment by anandbhaskaran** [comment]
I see the new PR is cooking! I would prefer to have smaller chunks going out rather than a big epic. What do you think?

**Comment by eudennis** [comment]
I don't see any activity on this repo, so I would imagine is not active anymore :(

**Comment by deafmute1** [comment]
give the pending PRs and community interest, would be cool if someone with more knowledge of JS and obsidian then me forked this :)

**Comment by elias-code** [comment]
Would love to see more development too. I think it is the best approach to obsidian todoist sync to this day!

**Comment by eudennis** [comment]
I made a fork, still in "alpha", but I'm planning to give it some effort. If anyone wants to take a look/suggestion.

https://github.com/eudennis/ultimate-todoist-sync-for-obsidian-experiment

**Comment by HeroBlackInk** [comment]
Yes! The project is active. v2.0.0 has just been released with bug fixes, a new Excluded Folders feature, improved sync defaults, and updated documentation. Check out the [CHANGELOG](https://github.com/HeroBlackInk/ultimate-todoist-sync-for-obsidian/blob/master/CHANGELOG.md) for details.
